pyproject.toml

```diff
@@ -14,7 +14,7 @@ dependencies = [
   "astropy",
   "numpy",
   "filelock",
-  "asdf",
+  "asdf @ git+https://github.com/braingram/asdf.git@diff_regression",
```

Comment:
This is temporary and will be removed when https://github.com/asdf-format/asdf/pull/1672 is merged and released and replaced with a specification that skips `3.0.0` but allows `>=3.0.1`

Reply:
`3.0.1` is released. I updated this to `!=3.0.0` to specifically not allow asdf `3.0.0` (but allow `3.0.1` and later versions).